rest/ruleset.xml

```diff
@@ -17,9 +17,6 @@
   <rule ref="rulesets/java/braces.xml/IfStmtsMustUseBraces">
     <priority>3</priority>
   </rule>
-  <rule ref="rulesets/java/unnecessary.xml/UnnecessaryModifier">
-    <priority>3</priority>
-  </rule>
```

Comment:
sneaky :)

Reply:
Oh that got in by accident, don't ask, eclipse...